src/main/scala/org/phenoscape/kb/PresenceAbsenceOfStructure.scala

```diff
@@ -67,6 +67,9 @@ object PresenceAbsenceOfStructure {
       model <- App.executeSPARQLConstructQuery(query)
     } yield {
       val dataset = new DataSet()
+      KB.buildDate.map {
+        date => dataset.setCurators("The author of this ontotrace matrix is Phenoscape KB built on " + date.toString)
```

Comment:
This is really dangerous—you should mostly never perform side effects inside a map. Depending how long `KB.buildDate` takes to complete, this will work some or most of the time but not always.